packages/agent/src/display.ts

```diff
@@ -452,3 +452,9 @@ export function getCredentialForDisplay(credentialRecord: W3cCredentialRecord |
     claimFormat: credentialRecord.credential.claimFormat,
   }
 }
+
+export const getReadableNameFromHost = (host: string) => {
+  const parts = host.split('.')
+  const domainName = parts.length >= 2 ? parts[parts.length - 2] : host
```

Comment:
Feels dangerous. E.g. animosolutions.azurewebsites.com will result in azurewebsites i think?

Reply:
Also removed.